### PR TITLE
Add support to initialize KojiWrapperBase from session

### PR DESCRIPTION
**Pull request by yazug**
- add unittest showing use case
- add check in KojiWrapperBase to initialize from session

This allow you to take an existing KojiWrapper object and
construct another KojiWrapperBase derivitve with only
session parameter provided

some_existing_wrapper

new_wrapper = KojiWrapper(session=some_existing_wrapper)

and the resulting wrapper created by this has all content initialized
correctly.

Closes: Issue #28

### CODE REVIEW

File: koji_wrapper/base.py [hunk 1]
```diff
@@ -16,6 +16,13 @@ def __init__(self, url='', topurl='', session=None):
         self.topurl = topurl
         self.session = session
 
+        if isinstance(session, KojiWrapperBase):
```
Comment (jguiditta) [inline koji_wrapper/base.py]: I think this should just go in the session setter, we already have logic here, just not the setting of url/topurl.  Otherwise, seems reasonable

File: tests/unit/test_koji_wrapper.py [hunk 2]
```diff
@@ -62,6 +62,20 @@ def test_init_with_koji_wrapper():
     assert test_wrapper.session is my_koji_wrapper.session
 
 
+def test_init_with_koji_wrapper_only():
+    """
+    GIVEN we have a valid KojiWrapper with a session,
+    WHEN a KojiWrapper object is passed as the session to a new KojiWrapper only
+    THEN this object's session is set to the session and url/topurl of the passed object
+    """
+    my_koji_wrapper = build_wrapper(None)
```
Comment (jguiditta) [inline tests/unit/test_koji_wrapper.py]: To properly test the new behavior, we probably also need to change build_wrapper to take url and topurl params so we can start of with them empty for the test here

File: tests/unit/test_koji_wrapper.py [hunk 3]
```diff
@@ -62,6 +62,20 @@ def test_init_with_koji_wrapper():
...
+    test_wrapper = KojiWrapper(session=my_koji_wrapper)
```
Comment (yazug) [inline tests/unit/test_koji_wrapper.py]: I think your idea of changing build_wrapper is a good general improvement, it's not needed to test this specific test case as you can see in this line test_wrapper is what is getting a new KojiWrapper that's initialized cleanly from session only.  I'd recommend any refactor to improve the unittests be a separate commit.

File: tests/unit/test_koji_wrapper.py [hunk 4]
```diff
@@ -62,6 +62,20 @@ def test_init_with_koji_wrapper():
     assert test_wrapper.session is my_koji_wrapper.session
 
 
+def test_init_with_koji_wrapper_only():
+    """
+    GIVEN we have a valid KojiWrapper with a session,
+    WHEN a KojiWrapper object is passed as the session to a new KojiWrapper only
+    THEN this object's session is set to the session and url/topurl of the passed object
```
Comment (yazug) [inline tests/unit/test_koji_wrapper.py]: long lines here tripping up pep8
